content/docs/tutorials/scraping_dynamic_content.md

```diff
@@ -99,7 +99,8 @@ Now we simply pass it to `waitForFunction`:
 
 ```javascript
 // In Puppeteer you need to inject JQuery with
-// await Apify.utils.Puppeteer.injectJQuery(page);
+// await puppeteerUtils.injectJQuery(page);
+// imported from 'crawlee' package
```

Comment:
fyi we also have `utils.puppeteer.*` in the crawlee package

Reply:
But it's imported only from meta-package, right? I mean - it would work here, but in the end people will opt for specific packages I guess. But I am not sure, I could keep the `utils.puppeteer` - it's kinda a matter of preference I'd say